pkg/kubelet/kubelet.go

```diff
@@ -1661,24 +1667,24 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 			if mirrorPod.DeletionTimestamp != nil || !kl.podManager.IsMirrorPodOf(mirrorPod, pod) {
 				// The mirror pod is semantically different from the static pod. Remove
 				// it. The mirror pod will get recreated later.
-				klog.Infof("Trying to delete pod %s %v", podFullName, mirrorPod.ObjectMeta.UID)
+				klog.InfoS("Trying to delete pod", "pod", klog.KObj(pod), "podUID", mirrorPod.ObjectMeta.UID)
 				var err error
 				deleted, err = kl.podManager.DeleteMirrorPod(podFullName, &mirrorPod.ObjectMeta.UID)
 				if deleted {
-					klog.Warningf("Deleted mirror pod %q because it is outdated", format.Pod(mirrorPod))
+					klog.InfoS("Deleted mirror pod because it is outdated", "pod", klog.KObj(mirroPod))
```

Comment:
```suggestion
					klog.InfoS("Deleted mirror pod because it is outdated", "pod", klog.KObj(mirrorPod))
```

is mirrorPod a *v1.Pod? If not we might need to use KRef

Reply:
> is mirrorPod a *v1.Pod? 

yes
https://github.com/kubernetes/kubernetes/blob/59012258cad28d22cfa9c53b815f0f842f539978/pkg/kubelet/pod_workers.go#L60